src/test/java/com/microsoft/sqlserver/jdbc/SharedTimerTest.java

```diff
@@ -22,6 +24,11 @@ void getTimer() throws InterruptedException, ExecutionException, TimeoutExceptio
             CompletableFuture.allOf(futures.toArray(new CompletableFuture[0])).get(2, TimeUnit.MINUTES);
         } finally {
             executor.shutdown();
+            if (!executor.awaitTermination(800, TimeUnit.MILLISECONDS)) {
+                executor.shutdownNow();
```

Comment:
this seems really short. We tests in multiple environments including Azure DB which may require longer so this looks like it would cause intermittent failures. A quick search in other tests show that we wait 5 seconds there. We would also need to run full tests in our test lab to confirm it's ok in this case but pretty sure 800 ms would be too short

Reply:
Hi @lilgreenbird, since I don't have the experiment env for the Azure DB, I have only test it on my local machine. As you suggested, I changed it to 5000ms, I will leave a comment in the test to mention that this may needs some confirmation with the set lab.

Reply:
Thanks for the change, we'll run these changes and see if the timeout value works for our test lab.